.github/actions/api-sdk-check/action.yml

```diff
@@ -20,7 +20,7 @@ runs:
     - name: Generate API SDK
       shell: bash
       run: |
-        sh scripts/build-api-sdk.sh
+        bash scripts/build-api-sdk.sh
```

Comment:
to fix this https://github.com/recallnet/js-recall/actions/runs/16812830870/job/47622181916#step:9:83